### DISCUSSION TITLE
r_max

### DISCUSSION THREAD

**Discussion by terryfrankcombe** [mir-group/nequip#411] [General]
A silly question: is r_max a cutoff in the normal force field sense... the longest range interaction that is modelled?  I.e. if two molecules are more than r_max apart then relative motion does not change the energy?

**Comment by Linux-cpp-lisp** [comment]
Hi @terryfrankcombe ,

This depends on the choice of model.  `r_max` determines the cutoff for the neighborlist.  As a message-passing graph neural network, NequIP does not restrict interactions to this range, as information is propagated along the graph.   Allegro, on the other hand, is specifically designed to avoid this property of the various message-passing equivariant networks, and is strictly local to the user's chosen `r_max`.
